.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,10 @@
 <!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->
 
+<!-- 
+NOTE: Replace "current" with a future version of Astro if this PR is for an upcoming release (e.g. a feature not yet released) and include a link to the `withastro/astro` implementation PR.
+-->
+### For Astro version: `current` <!-- [<`withastro/astro` implementation PR>](<url>) -->
```

Comment:
```suggestion
<!-- 
NOTE: If this PR is for an upcoming release (e.g. a feature not yet released), uncomment the line below. Replace `version` with a future version of Astro (e.g. `2.8.0`) and include a link to the `withastro/astro` implementation PR.
-->

<!-- ### For Astro version: `version`. See [Astro PR #](url). -->
```
I think it makes more sense to state the "if" first, so that if not, people just ignore it!

What do you think about this? I think maybe starting with the line entirely hidden, and just letting the feature dev's uncomment it is actually MORE helpful to me, because I won't see as many "mistakes" and will ONLY see this line if I have to worry about not merging immediatly.

Reply:
I'm going to commit to see if we get the Netlify checks to pass...

Reply:
I agree with both things. the user will simply ignore it as you mention as its PR is not of that type. and leaving the title commented by default will confuse the users even less and will be much more readable for you as you mentioned.